auth-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth-web",
-  "version": "2.6.128",
+  "version": "2.6.129",
```

Comment:
If this is the beginning of the end for the MBR, maybe we should finally go to 2.7.0? 😆